src/Switch.js

```diff
@@ -1,21 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import cx from 'classnames';
 import idgen from './idgen';
 
-const Switch = ({ checked, onChange, disabled, id, offLabel, onLabel }) => {
-  const computedId = id || idgen();
-
+const Switch = ({
+  id,
+  checked,
+  onChange,
+  disabled,
+  offLabel,
+  onLabel,
+  ...props
+}) => {
   return (
     <div className="switch">
-      <label htmlFor={computedId}>
+      <label htmlFor={id}>
         {offLabel}
         <input
-          id={computedId}
+          id={id}
           disabled={disabled}
           onChange={onChange}
           type="checkbox"
-          defaultChecked={checked}
+          checked={checked}
```

Comment:
since we have spread, do we need `id, disabled, onChange, checked`

Reply:
Yes, we have to keep these `prop`s as is